src/forklift/models.py

```diff
@@ -425,10 +427,10 @@ def get_adds_where_clause(self, source_primary_key, key_type, temp_suffix):
         temp_suffix string the suffix appended to forklift temp data
 
         return sql in clause if table is source table or return None if temp table'''
```

Comment:
i think we might add the the statement to use none for the temp table as well but it might need a bit more review.